workflows/ci.yml

```
@@ -48,7 +48,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
       - name: Run post-conditions
-        run: test -f .github/actions.yml && eval "$(yq r .github/actions.yml -D "true" post.ci)" || true
+        run: test -f .github/actions.yml && eval "$(yq -r .post.ci .github/actions.yml)" || true
```

Taking a look at the [`v3 to v4`](https://mikefarah.gitbook.io/yq/upgrading-from-v3#reading-with-default-value) docs, we should use and "evaluate" expression and recover the default value:

```suggestion
        run: test -f .github/actions.yml && eval "$(yq e '.post.ci // "true"' .github/actions.yml)" || true
```

---

Fixed